corehq/apps/cloudcare/tests/test_api.py

```diff
@@ -304,13 +303,13 @@ def _create_case(user, type, close=False, **extras):
         user_id=user.user_id,
         owner_id=user.user_id,
         **extras
-    ).as_xml(format_datetime=date_to_xml_string)]
```

Comment:
I'm not really sure why this was here, and removing it might mean that something else is broken?